client/components/deposits-overview/index.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { useAllDepositsOverviews } from 'wcpay/data';
+import strings from './strings';
+
+interface OverviewProps {
+	overview: AccountOverview.Overview | null | undefined;
+	account: AccountOverview.Account | null | undefined;
+	isLoading: boolean;
+}
+
+/**
+ * Renders a deposits overview
+ *
+ * @param {OverviewProps} props Deposits overview
+ * @return {JSX.Element} Rendered element with deposits overview
+ */
+const DepositsOverviewDetails: React.FunctionComponent< OverviewProps > = (
+	props
+) => {
+	return (
+		<Card>
+			<CardHeader>{ strings.heading }</CardHeader>
+
+			<p>Next Deposits Section Goes here</p>
+
+			<p>Deposits History Section Goes here</p>
+
+			<p>Deposits Card Footer/Action Goes here</p>
+		</Card>
+	);
+};
+
+const DepositsOverview = (): JSX.Element => {
+	const {
+		overviews,
+		isLoading,
+	} = useAllDepositsOverviews() as AccountOverview.OverviewsResponse;
+
+	const { currencies, account } = overviews;
+
+	if ( isLoading ) {
+		return (
+			<DepositsOverviewDetails
```

Comment:
A note for the future:

When we have the UI finalised, we should create a separate component that is rendered only while `isLoading` is true, using the `<Loadable />` component for the "loading" animation UI.

For example: https://github.com/Automattic/woocommerce-payments/blob/8459303f34be76f2e563134f32dac3e09a8e3a7b/client/components/deposits-information/loading.tsx

Reply:
I wasn't actually sure if we'd need to do something different here because the designs for the isLoading state only include `<Loadable />` components for the Next Deposits heading, description, and also the Next deposits table (screenshot below), so I was thinking the Next Deposits child component would have a loading component since it will have all of the styling needed for the table etc.

![image](https://user-images.githubusercontent.com/2275145/224897842-59239827-8608-46db-a9ca-bf0e55ac6c13.png)

Reply:
Ah yeh, I see, thanks! Pass the isLoading state to those components to decide 💯

Reply:
Yeah that's what I was thinking but I'm not actually sure which is the correct approach for this sort of thing. If you're okay with it then I guess we can go with this and iterate on it when we get to it